src/components/Auth.js

```diff
@@ -7,96 +7,89 @@ import './Auth.css';
-    const [ pwSuggestions, setPwSuggestions ] = useState([]);
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [pwSuggestions, setPwSuggestions] = useState([]);
 
     const handleEmail = (event) => {
-        setEmail(event.target.value)
-        console.log(email)
+        setEmail(event.target.value);
     };
 
     const handlePassword = (event) => {
         setPassword(event.target.value);
         const evaluation = zxcvbn(event.target.value);
-      
-        setPwSuggestions(evaluation.feedback.suggestions)
+        setPwSuggestions(evaluation.feedback.suggestions);
     };
 
-    const handleClick = (event) => {
-        setRegister(!register)
-    }
-
-    const handleSubmit = async (e) => {
-        e.preventDefault();
-
+    const handleRegister = async () => {
         try {
-            if (register){
-               const { data, error } = await supabase.auth.signUp({
-                    email,
-                    password,
-                })
-                console.log("data from signUp:", data);
-
-                if (error){
-                    console.log('error', error.message);
-                    return alert(error.message)
-                }
-
-                const { user } = data;
-                const { id, user_metadata } = user;
-                
-                if (id && !user_metadata.email_verified) {
-                    return alert('Please check your email to confirm your account')
-                }
-            } else {
-                const { data, error } = await supabase.auth.signInWithPassword({
-                    email,
-                    password,
-                });
-
-                console.log("data from signInWithPassword:", data);
-
-                if (error){
-                    return alert("Error in signing in. If you've already signed up, check your email for a confirmation email.")
-                }
-                getUserSession();
+            const { data, error } = await supabase.auth.signUp({
+                email,
+                password,
+            });
+
+            if (error) {
+                console.log('error', error.message);
+                return alert(error.message);
+            }
 
+            const { user } = data;
+            const { id, user_metadata } = user;
+
+            if (id && !user_metadata.email_verified) {
+                return alert('Please check your email to confirm your account');
             }
-     
-        } catch (error) {
-            console.log('FE signUp/signIn Error: ', error);
 
+            await getUserSession();
+        } catch (error) {
+            console.log('Error in signUp:', error);
             setEmail('');
             setPassword('');
         }
-
     };
 
-        const getUserSession = async () => {
+    const handleLogin = async () => {
+        try {
+            
+            const { error } = await supabase.auth.signIn({
```

Comment:
@rachelspencer I suspect if you change this back to `supabase.auth.signInWithPassword()` it might fix the issue you're having

Reply:
I changed this back!